crawler.py

```diff
@@ -27,6 +27,7 @@
 TIMEOUT = int(os.getenv('TIMEOUT', '60'))
 SENTRY_DSN = os.getenv('SENTRY_DSN', None)
 LOGGER_SETTINGS = MPLogger.parse_config_from_env()
+MAX_JOB_RETRIES = int(os.getenv('MAX_JOB_RETRIES', '3'))
```

Comment:
(Dupe comment to make sure it touches all instances of "3"): Note: [Scrapy has a default of 2](https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#retry-times)